docs/docs/animations/withTiming.mdx

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+---
+
+# withTiming
+
+`withTiming` lets you create an animation based on duration and [easing](https://easings.net/).
+
+import TimingBasic from '@site/src/examples/TimingBasic';
+import TimingBasicSrc from '!!raw-loader!@site/src/examples/TimingBasic';
+
+<InteractiveExample src={TimingBasicSrc} component={<TimingBasic />} />
+
+## Reference
+
+```javascript
+import { withTiming } from 'react-native-reanimated';
+
+function App() {
+  sv.value = withTiming(0);
+  // ...
+}
+```
+
+<details>
+<summary>Type definitions</summary>
+
+```typescript
+type AnimatableValue = number | string | number[];
+
+interface WithTimingConfig {
+  duration?: number;
+  easing?: EasingFunction;
+}
+
+function withTiming<T extends AnimatableValue>(
+  toValue: T,
+  config?: WithTimingConfig,
+  callback?: (finished?: boolean, current?: AnimatableValue) => void
+): T;
+```
+
+</details>
+
+### Arguments
+
+#### `toValue`
+
+The value on which the animation will come at rest.
+
+#### `config` <Optional />
+
+The timing animation configuration.
+
+import { useTimingPlayground } from '@site/src/components/InteractivePlayground';
+
+<InteractivePlayground usePlayground={useTimingPlayground} />
+
+Available properties:
+
+| Name                  | Type     | Default                     | Description                                           |
+| --------------------- | -------- | --------------------------- | ----------------------------------------------------- |
+| duration <Optional /> | `number` | 300                         | Length of the animation (in milliseconds).            |
+| easing <Optional />   | `Easing` | `Easing.inOut(Easing.quad)` | An easing function which defines the animation curve. |
+
+##### `Easing`
+
+The `easing` parameter lets you fine-tine the animation over the specified time duration. For example, you can make the animation begin with fast acceleration and then slow it down towards the end, or start slowly then pick up some speed before slowing down again towards the end.
+
+It all will start to make sense when you compare side by side a `linear` easing with the default `Easing.inOut(Easing.quad)` easing.
```

Comment:
```suggestion
It will all start to make sense when you compare a `linear` easing side by side with the default `Easing.inOut(Easing.quad)` easing.
```